### DISCUSSION TITLE
How best to manage historical data?

### DISCUSSION THREAD

**Discussion by sameleff** [ghostfolio/ghostfolio#2609] [Q&A] [answered]
I see where you can import historical data for live datasources (e.g. YAHOO but not MANUAL) within the Admin Control panel, but I have two questions:
1. Will adding historical data from the "edit activity" GUI cause issues for live datasources such as YAHOO? Put another way, if I am using a live datasource, will I break the connection if I import historical data and paste it into the "edit activity" GUI in order to add it to Ghostfolio?
2. What is the difference between Gather Recent Data, Gather All Data, and Gather Profile Data? What is the definition of 'recent'?

**Comment by dtslvr** [comment, marked as answer]
Hi @sameleff

1. No, importing historical data manually should not interfere with the "live" data source such as `YAHOO`. However, it is possible that the data from Yahoo will overwrite your imported data.
2. _Gather Recent Data_ fetches historical data from the last 7 days while _Gather All Data_ fetches the full history and can take a while (depending on the number of your asset profiles). _Gather Profile Data_ in contrast fetches (meta) data related to your asset profiles (such as asset classes, countries, sectors, ...).
